vnext/Microsoft.ReactNative.Cxx/ReactPromise.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include "ReactError.h"
+#include "winrt/Microsoft.ReactNative.Bridge.h"
+
+namespace winrt::Microsoft::ReactNative::Bridge {
```

Comment:
>winrt [](start = 10, length = 5)

Do we use this namespace for things outside of cppwinrt? I immediately thought this type was part of that based on namespace when I saw it's first usage,

Reply:
I want to address the whole issue about namespaces as a separate PR. At this point it was just easier to keep the same namespace to avoid long names.

Reply:
Could you add a Github issue for this to RNW with the "JSHost" tag?

---
In reply to: [356845271](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356845271) [](ancestors = 356845271)